edumanage/views.py

```diff
@@ -2654,3 +2654,10 @@ def settings_dict_get(setting, *keys, **opts):
     if dct == {}:
         dct = opts.get('default', None)
     return dct
+
+# utility function to merge two dictionaries
+def merge_dicts(dict1, dict2):
```

Comment:
This could be moved to a more general utility module.
Also, could potentially be simplified to `return dict(itertools.chain(dict1.items(), dict2.items()))` (requires importing itertools)

Reply:
Agreed with @cangus to keep this as-is - there are already other utility functions at the end of `edumanage/views.py` and as `merge_dicts` is only used from within this module, it makes the code more readable this way.

Reply:
It's not a big deal but I would probably opt for changing `base_response()` to accept a dict as a second argument, to be merged with the context it produces.

Also +1 for `itertools.chain()`.